util/emailUtil.js

```diff
@@ -0,0 +1,53 @@
+const nodemailer = require('nodemailer');
+
+const AppConfig = require('../config').AppConfig;
+
+// CC recipients are ones included in the email, but not intended directly for them 
+// in this case this will be the (super)admins of the company
+/**
+ * 
+ * @param {string} subject 
+ * @param {string} content 
+ * @param {string[]} recipients 
+ * @param {string[]} ccRecipients 
+ */
+const sendEmail = (subject, content, recipients, ccRecipients) => {
+
+    if (!recipients || !subject || !content) {
+        return false;
+    }
+
+    const transporter = nodemailer.createTransport({
+        service: 'gmail',  // could be moved to config.js
+        secure: true,
+        port: 587,
+        auth: {
+            user: AppConfig.EMAIL,
+            pass: AppConfig.EMAIL_PASSWORD
+        }
+    });
+
+    transporter.verify().then(success => {
+
+        const message = {
+            from: AppConfig.EMAIL,
+            to: recipients,
```

Comment:
Should we validate recipients list to check if the list contains valid emails, I don't know what will happen if one of them is invalid and others are valid, did you check that case?

Reply:
There is a `rejected` field in the response object return from `sendMail`. I think that invalid ones are classified to that `rejected` array.

Reply:
We could use [validator](https://www.npmjs.com/package/validator)  for that purpose. 
It's easy to use and it provides a lot of useful validation functions. 
For example:

```
import validator from 'validator';

validator.isEmail('test@mail.com');
```

Reply:
`validator` will return `true` for that email, but it is a non-existent mail, so it will be in the `rejected` array as well. As such, I don't think that checking it is necessary.

Reply:
@SavoX2 if the module doesn't have an internal email validation we should add it. Why should the server try to send emails which aren't valid? We can't make sure that all emails in the recipient list are existent, but at least we can filter those which aren't valid. Besides that. removing invalid emails before sending would benefit in less network traffic

Reply:
You already have that inside the `User` model(schema) and the field is `immutable`. To doint another check here would be redundant.